Casks/p/processspy.rb

```diff
@@ -0,0 +1,23 @@
+cask "processspy" do
+  version "1.6.0"
+  sha256 "a7426398bcd5e2c5b7d72b3288f5d74c63e12179e2eceb14a6a90d935ff3f8bd"
+
+  url "https://process-spy.app/archive/ProcessSpy_#{version}.dmg"
+  name "ProcessSpy"
+  desc "Monitor processes with command-line arguments"
```

Comment:
```suggestion
  desc "Process monitor"
```

A suggestion to keep it concise (see [documentation](https://docs.brew.sh/Cask-Cookbook#stanza-desc)).